lingua-franca-ref.txt

```diff
@@ -1 +1 @@
-master
+47b87a49ceece1f0065a9dbf9b8a06c9fcdfe01d
```

Comment:
```suggestion
network-structure
```

Reply:
Please make sure to test against the latest in commit in the `network-structure` branch.